src/main/java/app/gpx_animator/core/util/PointUtil.java

```diff
@@ -0,0 +1,49 @@
+package app.gpx_animator.core.util;
```

Comment:
Please add the copyright and license header to the file. Just copy it from the beginning of any other source file.

Reply:
fixed add  Copyright for PointUtil class

Reply:
I fixed problem and add test